apps/docs/content/components/Prose.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Prose
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/Prose/Prose.tsx
+storybook: '/brand/storybook/?path=/story/components-prose--default'
+description: Use the Prose component to render html content with Primer Brand styles.
+---
+
+```js
+import {Prose} from '@primer/react-brand'
+```
+
+## Examples
+
+<Note>
+
+`Prose` is designed to take sanitized HTML as a string and render it with Primer Brand styles. It is not intended to be used with JSX. The `Prose` component doens't sanitize the data internally. It is the responsibility of the consumer to sanitize the data before passing it to the component.
+
+</Note>
+
+### Default
+
+```jsx live
+<Prose
+  rawHtmlMarkup={`
+	<h1>The tools you need to build what you want.</h1>
+    <p>Contribute to projects quickly with automatic environment setup.</p>
+    <p>Make sure you see the changes you care about.</p>
+    <p>Build community around your code.</p>
+
+    <img
+      src="https://via.placeholder.com/600x400/f5f5f5/f5f5f5.png"
+      alt="placeholder, blank area with an off-white background color"
+    />
+
+    <h2>Automation and CI/CD</h2>
+    <ul>
+      <li>
+        Automate everything: CI/CD, testing, planning, project management, issue labeling, approvals, onboarding, and
+        more.
+      </li>
+      <li>Standardize and scale best practices, security, and compliance across your organization.</li>
+      <li>Get started quickly with thousands of actions from partners and the community.</li>
+    </ul>
+
+    <h2>Security</h2>
+    <p>
```

Comment:
General comment about `p`, but unrelated to this line. Could our Prose handle styles for `kbd` tags too?

E.g. from GitHub Blog

<img width="411" alt="Screenshot 2023-06-26 at 12 30 05" src="https://github.com/primer/brand/assets/13340707/0ba08d0b-3af9-48ea-9021-31903eca2e33">

As this could technically be a new component too, also fine to push this to a later Phase if easier.

Reply:
As we discussed in maintainers sync we are going to push this to a follow-up ticket.